strongbox-web-core/pom.xml

```diff
@@ -906,7 +906,18 @@
                                     <name>strongbox.truststore.jks</name>
                                     <value>${dir.strongbox.home}/etc/ssl/truststore.jks</value>
                                 </systemProperty>
-
+                                <systemProperty>
+                                    <name>orientdb.profile</name>
```

Comment:
Could you please prefix this with `strongbox.*`?

Reply:
ok

Reply:
done